package.json

```diff
@@ -18,15 +18,15 @@
     "build_es6": "rm -rf dist/es6 && tsc src/Rx.ts src/Rx.KitchenSink.ts --outDir dist/es6 --sourceMap --target ES6 -d --diagnostics",
     "build_closure": "java -jar ./node_modules/google-closure-compiler/compiler.jar ./dist/global/Rx.js --create_source_map ./dist/global/Rx.min.js.map --js_output_file ./dist/global/Rx.min.js",
     "build_global": "rm -rf dist/global && mkdir \"dist/global\" && browserify src/Rx.global.js --outfile dist/global/Rx.js && npm run build_closure",
-    "build_perf": "npm run build_es6 && npm run build_cjs && npm run build_global && webdriver-manager update && npm run perf",
-    "build_test": "rm -rf dist/ && npm run lint && npm run build_es6 && npm run build_cjs && jasmine",
-    "build_cover": "rm -rf dist/ && npm run lint && npm run build_es6 && npm run build_cjs && npm run cover",
+    "build_perf": "npm run build_cjs && npm run build_global && webdriver-manager update && npm run perf",
+    "build_test": "rm -rf dist/ && npm run lint && npm run build_cjs && jasmine",
```

Comment:
Now cjs does not depends on es6 build results, removed some of tasks has chained build jobs for fast turnaround on dev machines. Travis still runs build_all.